src/components/SkillsPanel.js

```diff
@@ -72,6 +72,15 @@ export default function SkillsPanel({ characterStats }) {
 }
 
 function SkillTile({ skill, selectedSkill, setSelectedSkill, unlockedSkills, level = 1 }) {
+    const dispatch = useDispatch();
+    const isSkillUnlocked = unlockedSkills.includes(skill);
+    const isDefaultSkill = DEFAULT_UNLOCKED_SKILLS.includes(skill);
+
+    useEffect(() => {
+        if (level > 1 && !isSkillUnlocked) dispatch(unlockSkill({ skill }));
+        if (level === 1 && isSkillUnlocked && !isDefaultSkill) dispatch(lockSkill({ skill }));
```

Comment:
thinking about this a bit. I like auto-unlocking it if we notice they've leveled it up, but I'm not sure about locking it for them if it's level 1. Could be that they just aren't high enough to get on hiscores yet (I think you have to get to 30 or something).

Reply:
Reason i did this was in the case you look up multiple/new users. If you don't lock the skill if it's level 1, it will appear unlocked as level 1 just because the last account you searched had the skill unlocked (>1).

Without a proper flag it's impossible to know, so hopefully we might be able to get that eventually!

Reply:
Ahh ok gotcha. Yeah right now everything assumes you are only using one character. I'll be adding multiple character support with the backend stuff (and probably remove the input box for hiscores so it'll just lookup whichever character you have set as your active one). I think it's fine to assume only one character right now.